src/test/java/com/squareup/kotlinpoet/KotlinPoetTest.kt

```diff
@@ -281,4 +281,36 @@ class KotlinPoetTest {
         TypeName.get(Int::class).asNullable()).asNullable()
     assertThat(list.toString()).isEqualTo("java.util.List<kotlin.Int?>?")
   }
+
+  @Test fun getAndSet() {
+    val source = KotlinFile.builder(tacosPackage, "Taco")
+        .addProperty(PropertySpec.varBuilder("propertyWithCustomAccessors", Int::class)
+            .initializer("%L", 1)
+            .getter(FunSpec.getterBuilder()
+                .addStatement("println(%S)", "getter")
+                .addStatement("return field")
+                .build())
+            .setter(FunSpec.setterBuilder()
+                .addParameter("value", Int::class)
+                .addStatement("println(%S)", "setter")
+                .addStatement("field = value")
+                .build())
+            .build())
+        .build()
+    assertThat(source.toString()).isEqualTo("""
+        |package com.squareup.tacos
+        |
+        |import kotlin.Int
+        |
+        |var propertyWithCustomAccessors: Int = 1
+        |  get() {
```

Comment:
I think we start with full verbose syntax, then later tweak it to elide stuff we can elide